frontend/js/src/lastfm/LastFMImporter.tsx

```diff
@@ -321,6 +321,31 @@ export default class LastFmImporter extends React.Component<
     this.startImport();
   };
 
+  submitSpotifyStreams = async (text: string) => {
+    const streams: Array<SpotifyStream> = JSON.parse(text);
+    const listens = streams
+      .filter(
+        (stream) =>
+          stream.ms_played > 30000 &&
```

Comment:
According to the [ListenBrainz documentation](https://listenbrainz.readthedocs.io/en/latest/users/api/core.html?post--1-submit-listens#post--1-submit-listens) a listen should only be submitted if it was played for more than half the track or for at least 4 minutes.